website/content/docs/api-clients/client-agent.mdx

```diff
@@ -274,9 +274,15 @@ If you want to disable the Boundary Client Agent, you can stop it with the follo
 <Tab heading="MacOS" group="macos">
 
 ```shell-session
-$ sudo launchctl stop com.hashicorp.boundary.boundary-client-agent
+$ sudo launchctl unload -w /Library/LaunchDaemons/com.hashicorp.boundary.boundary-client-agent.plist
 ```
 
+<Note>
+
+Unloading the Boundary client agent removes its launch daemon configuration. To restart the agent, use `sudo launchctl load ...`
```

Comment:
I think we have enough space here to show the whole command, to make it easier for the user:

Unloading the Boundary client agent removes its launch daemon configuration. To restart the agent, use:

```shell-session
 $ sudo launchctl load -w /Library/LaunchDaemons/com.hashicorp.boundary.boundary-client-agent.plist
```

Reply:
I initially tried wrapping the code snippet under the Note, but it overflowed and didn't display properly—I think the Note tag doesn't support wrapping code snippets. So I ellipsised it (...) to keep things concise. I also considered adding the entire command as an inline code snippet, but it looked too congested.

<img width="964" alt="image" src="https://github.com/user-attachments/assets/06e3be12-dfe1-4b0e-b868-05c1f58020fc">

Reply:
Thank you for checking, that does indeed look very odd. Maybe we should remove the "Note" wrapper and just include the text and the command? That should still look ok, I think?

Reply:
I’ve updated the code as suggested. The "Note" wrapper has been removed, and the text now directly includes the explanation alongside the command.